### PR TITLE
Disabling tensorboard logging and config saving

### PR DESCRIPTION
**Pull request by jordis-ai2**
- Note: if machine params contain a visualizer, visualization info is still collected by the inference engine. Is this sufficient, even if slightly inefficient or I should check in the inference engine if we are logging to tensorboard before enabling the visualizer?

### CODE REVIEW

File: core/algorithms/onpolicy_sync/runner.py [hunk 1]
```diff
@@ -420,6 +427,9 @@ def metric_path(self, start_time_str: str) -> str:
         )
 
     def save_project_state(self):
+        if self.disable_config_saving:
+            return
+
```
Comment (Lucaweihs) [inline core/algorithms/onpolicy_sync/runner.py:432]: Can we instead do something like:
```python
if not self.disable_config_saving:
    self.save_project_state()
```
wherever `save_project_state` is called? I know it's unlikely but I could imagine someone not wanting configs to save during training but perhaps having some manual call to `save_project_state` in some script.

**Review by Lucaweihs — COMMENTED** [review]
Looks good! One minor comment, feel free to merge after addressing.

File: core/algorithms/onpolicy_sync/runner.py (hunk 1 above)
Comment (jordis-ai2) [reply core/algorithms/onpolicy_sync/runner.py:432]: Sure, I'm changing it now 👍

**Comment by jordis-ai2** [comment]
Merging after addressing review.

**Pull request merged**